docs/UserGuide.adoc

```diff
@@ -232,14 +312,60 @@ Format: `exit`
 
 === Saving the data
 
-Organizer data are saved in the hard disk automatically after any command that changes the data. +
+PrioriTask data is saved in the hard disk automatically after any command that changes the data. +
 There is no need to save manually.
 
-// tag::dataencryption[]
-=== Encrypting data files `[coming in v2.0]`
+=== Deleting specific parameters from a task `[coming in v2.0]`
+
+_{ coming in v2.0 }_
+
+****
+* Delete either a tag, deadline, or description from a particular task.
+****
+
+=== Locating tasks by deadline `[coming in v2.0]`
+
+_{ coming in v2.0 }_
+
+=== Locating tasks by tag `[coming in v2.0]`
+
+_{ coming in v2.0 }_
+
+=== Locating tasks by description `[coming in v2.0]`
```

Comment:
Isn't locating task by description done by the `finddes` command stated above? =p